Rakefile

```
@@ -1,37 +1,66 @@
-
 sfe_root = ENV['SFE_ROOT']
+extension_name = ENV['SFE_NAME']
 
 unless sfe_root
   abort "SFE_ROOT not set"
-end
+else
+  extensions_dir = File.join(sfe_root, "extensions")
+  extension_dir = File.join(extensions_dir, extension_name)
+  install_lib_dir = File.join(extensions_dir, "__lib__")
 
-desc "Compile, build and install jar"
-task :default => "rubyfox"
+  unless File.directory?(install_lib_dir)
+    FileUtils.mkdir_p(install_lib_dir)
```

I *think* `FileUtils` is [included by default](https://github.com/ruby/rake/blob/8d3c5492b86567d8915e0fbea8597582d1fad876/lib/rake/dsl_definition.rb#L11-L12) in `Rakefile` so we could drop `FileUtils.` everywhere.

```suggestion
    mkdir_p(install_lib_dir)
```

---

Good catch, thanks for the review.